src/components/SuperDashListCard/SuperDashListCard.test.tsx

```diff
@@ -12,6 +12,8 @@ describe('Testing the Super Dash List', () => {
         orgName="Dogs Care"
         orgLocation="India"
         createdDate="04/07/2019"
+        admins="23"
+        members="123"
```

Comment:
is this change really needed if yes why?

Reply:
Yes, these are needed because the motive of this pull request is to remove the hardcoded values on the organization list page, and in hardcoded values, admins count and members are also there.

To resolve this, I need to add admin and member count values dynamically.

Reply:
yeh dynamical nahi h shyd you are testing it for the hardcoded problem i will love to see a way where we can test the dynamical data by dynamical test data

Reply:
@yasharth291 I have made the changes.